tests/server/blueprints/cases/test_matchmaker_controllers.py

```diff
@@ -102,7 +162,8 @@ def test_matchmaker_add(app, user_obj, case_obj, test_hpo_terms, mocker):
 
         # THEN ine case should be uodated and a submission object correctly saved to database
         updated_case = store.case_collection.find_one()
-        assert len(updated_case["mme_submission"]["patients"]) == 1
+        submission = updated_case["mme_submission"]
+        assert submission["patients"][0]["genomicFeatures"][0]["gene"]["id"] == "TRAPPC9"
```

Comment:
This works ofcourse, but you could make that gene symbol a local constant (variable 😝) if nothing else to speed up reading. I guess fixture is over the top for single use.

Reply:
This is fixed now, I'm going to merge!